imports/ui/components/ideas/edit/IdeaEditSubmenu.jsx

```diff
@@ -89,11 +89,10 @@ export default class IdeaEditSubmenu extends Component {
             <button onClick={this.changeView} type="submit" className="waves-effect waves-light green btn right">
               <span className="fa fa-check-circle"></span> Редактировать
             </button> :
-            <BookmarkIdeaLink
```

Comment:
button will be rendered inside BookmarkIdeaLink component so it should stay here anyway